src/css/betterttv-dark.css

```diff
@@ -3650,14 +3650,20 @@ button.follow-button, .subscription-modal__right button, .subscription-modal__le
     box-shadow: inset 0 -1px 0 #333, inset -1px 0 0 #333, inset 1px 0 0 #333;
 }
 
+.activity-create--card,
 .activity-card {
     background-color: rgb(28, 28, 28);
     box-shadow: inset 0 0 0 1px #333;
     color: #d3d3d3;
 }
 
-.activity-card__comments, .activity-comments {
-    background-color: #282828;
+.activity-create--card {
+    padding: 1.5rem;
```

Comment:
I do not have a high dpi device with a development environment set up at the moment, so no, it is however the same padding that twitch uses in their post cards below.